Lib/test/test_shutil.py

```diff
@@ -141,6 +142,17 @@ def supports_file2file_sendfile():
 
 SUPPORTS_SENDFILE = supports_file2file_sendfile()
 
+# AIX 32-bit mode, by default, lacks enough memory for the xz/lzma compiler test
+# The AIX command 'dump -o program' gives XCOFF header information
+# The second word of the last line in the maxdata value
+# when 32-bit maxdata must be greater than 0x1000000 for the xz test to succeed
+def _maxdataOK():
+    if AIX and sys.maxsize == 2147483647:
+        hdrs=subprocess.getoutput("/usr/bin/dump -o %s" % sys.executable)
```

Comment:
Is /usr/bin/dump guaranteed to exist on AIX, or does there need to be a separate existence check here?

Reply:
/usr/bin/dump (symbolic link to /usr/sbin/dump) is installed by default, and cannot be removed using the package manager (installp).